### PR TITLE
fix: Uses dynamic payload type mapping for VP8 when probing for bandwidth.

### PR DESCRIPTION
**Pull request by gpolitis**

### CODE REVIEW

**Comment by gpolitis** [comment]
@bgrozev or @bbaldino could you please take a look?

File: src/main/java/org/jitsi/videobridge/cc/BandwidthProbing.java [hunk 1]
```diff
@@ -79,6 +80,11 @@
      */
     private final VideoChannel dest;
 
+    /**
+     * The VP8 payload type to use when probing with the SSRC of the bridge.
+     */
+    private int vp8PT = -1;
```
Comment (bbaldino) [inline src/main/java/org/jitsi/videobridge/cc/BandwidthProbing.java]: technically this should match whatever video PT that's being sent, is that right?  we're just hard-coding to vp8 for now?

Comment (gpolitis) [reply src/main/java/org/jitsi/videobridge/cc/BandwidthProbing.java]: Even if the conference runs over VP9, we can still send padding packets with VP8 payload type; as long as it's signaled, we're good. We're not sending any real media anyway, just padding (empty packets).

**Comment by bbaldino** [comment]
lgtm

**Pull request merged**